### PR TITLE
Prepare 2.x for OpenShift CI - part 4

### PR DESCRIPTION
**Pull request by wainersm**
This is the continuation of part 1 (https://github.com/kata-containers/tests/pull/3102), part 2 (https://github.com/kata-containers/tests/pull/3139)  and part 3 (https://github.com/kata-containers/tests/pull/3175) towards getting Kata Containers 2.x on-board into OpenShift CI.

The important fix is regarding set SELinux to permissive mode, with this I hope to have the last change in place to allow build, install and run a single smoke test on OpenShift CI. The other change is just a bonus.

### CODE REVIEW

**Review by snir911 — APPROVED** [review]
LGTM, Thanks!

**Review by fidencio — APPROVED** [review]
lgtm. thanks @wainersm!

**Comment by fidencio** [comment]
@wainersm, please, wait for a review of someone who's not a Red Hatter before getting this one merged. :-)

**Comment by wainersm** [comment]
> @wainersm, please, wait for a review of someone who's not a Red Hatter before getting this one merged. :-)

I had my finger on the "merge pull request" button. Now I hold on, take a breath... :)

**Review by chavafg — APPROVED** [review]

**Pull request merged**
